.github/workflows/lint.yml

```diff
@@ -57,25 +57,22 @@ jobs:
           echo "::add-matcher::.github/workflows/matchers/typos.json"
       - name: Install dependencies
         run: |
-          uv pip install --system -r requirements/lint-requirements.txt
-      - uses: ./.github/actions/show-versions
-      - uses: ./.github/actions/pipdeptree
+          uv sync --only-group lint
```

Comment:
you dont necessarily need to do this step since the below line will do it automatically